src/custom/state/claim/hooks/index.ts

```diff
@@ -62,7 +62,7 @@ import useIsMounted from 'hooks/useIsMounted'
 import { ChainId } from '@uniswap/sdk'
 import { ClaimInfo } from 'state/claim/reducer'
 
-const CLAIMS_REPO_BRANCH = 'gip-13'
```

Comment:
Very careful with merging this. If this gets to production we will have some troubles